src/WeekComponent/WeekComponent.css

```diff
@@ -6,76 +6,99 @@
 .description-week-day {
   height: 26px;
   align-self: center;
-  padding: 10px;
+  padding: 15px 178px 24px 0px;
   letter-spacing: 1.19px;
-  font-family: "San Francisco";
+  font-family: 'arial';
+  font-weight: bold;
 }
 
 .pause-start-button {
-  width: 159px;
-  height: 52px;
+  width: 336px;
```

Comment:
Better to use percentage here if your intention is to take the width of the screen.